cmd/integrationArtifactDeploy.go

```diff
@@ -99,7 +99,11 @@ func runIntegrationArtifactDeploy(config *integrationArtifactDeployOptions, tele
 		log.Entry().
 			WithField("IntegrationFlowID", config.IntegrationFlowID).
 			Info("successfully deployed into CPI runtime")
-		deploymentError := pollIFlowDeploymentStatus(retryCount, config, httpClient, serviceKey.OAuth.Host)
+		taskId, readErr := ioutil.ReadAll(deployResp.Body)
+		if readErr != nil {
+			return errors.Wrap(readErr, "HTTP response body could not be read")
```

Comment:
Would it make sense to make this a bit more descriptive- something that mentions the task id? Maybe something like "Task Id not found: HTTP response body could not be read"

Reply:
done